lib/bulk-apply-applications.js

```diff
@@ -170,17 +181,18 @@ function bulkApplyApplications (args, callback) {
     if (updateAction === 'disapprove' || updateAction === 'checkin' || !available) return done(null, applications);
```

Comment:
It seems like available is initialised as false and never updated. Should it not be ticketsAvailable that is used here?